lux/core/series.py

```diff
@@ -211,7 +215,83 @@ def recommendation(self):
             self.name = " "
         ldf = LuxDataFrame(self)
```

Comment:
Similar to what we did in frame.py, can we cache the result of the maintain_metadata and recs so that we do not have to create a new LuxDataFrame and regenerate the recommendations every time?

Reply:
yup fixed now!